### ISSUE TITLE
New idea - a "conversation" issue

### ISSUE THREAD

**Issue by SeburoSUMO** [chaoss/wg-diversity-inclusion#296]
Hi

I have been pointed in the direction of this project as a place that may be able to make use of an idea I have had further to my ongoing contribution to Mozilla.

Where contributors are involved in an open project on a platform such as GitHub, it is easy for issues to be taken up by conversation or discourse.  At the same time, I appreciate that there are some projects that do not have the resource to build and support a separate forum or chat platform, but they still wish to have open conversations with contributors.  Sometimes people may bring strong opinions that may not be relevant to software issues and "demand to be heard".

For these communities, would having a dedicated GitHub issue for community discourse be a useful recommended idea?  A GitHub issue is not a chat platform - I agree! - but as a way to contain discourse from "working" topics and have a dialogue with contributors in an open fashion, it might just work.

I appreciate that this idea has its flaws, but hopefully there might be some way you can use it and therefore have a fixed point in a repo where community dialogue can take place and possibly measured.

**Comment by GeorgLink** [comment]
Hi @SeburoSUMO -- I agree that each community can appropriate technology as it sees fit, including using a GitHub issue for conversations that are not software related.

What metric do you think this would best align with?

**Comment by SeburoSUMO** [comment]
Hi @GeorgLink 

Looking at the focus areas listed on the CHAOSS site, I would initially look at this as being linked to Communication Inclusivity, but at there is also an element of Governance in so much that having an easy to reach communication "bucket" such as this could be a good way for people to flag up urgent concerns.

**Comment by germonprez** [comment]
@SeburoSUMO is there anyway that you can proposed this via a shared document? This is something that I think we'd be happy to talk about at our next D&I call (July 15th).

**Comment by SeburoSUMO** [comment]
I hope that this is okay.  Document is set so that anyone with the link can comment:

https://docs.google.com/document/d/1NJ_h5iNCnefACvkkF0N__j_skhyBuAnMieDKwCSXyQg/edit?usp=sharing

**Comment by GeorgLink** [comment]
Yes, thank you!

At the bottom of the document, I copied the template we have for metrics so that we can start filling that out.

Could you please make the “edit” permission public? I was only able to make a suggestion.

**Comment by SeburoSUMO** [comment]
Hi @GeorgLink 

Sorry for that, I think I have now changed it for public editing:

https://docs.google.com/document/d/1NJ_h5iNCnefACvkkF0N__j_skhyBuAnMieDKwCSXyQg/edit?usp=sharing

**Comment by GeorgLink** [comment]
Thank you

**Comment by germonprez** [comment]
Hi @SeburoSUMO 

Could you take a look at the template that @GeorgLink provided in your doc and place some of your text into the appropriate headings? This would be great in helping us to consider this metric.

**Comment by SeburoSUMO** [comment]
Hi @germonprez 

Apologies for not responding earlier, I did not know there was a needinfo outstanding for me.

At the time I wrote that document, I was unaware of the [GitHub Discussions](https://docs.github.com/en/discussions) feature.  This is the same thing I was trying to outline - an area that contributors to a project without an open communication channel could discuss matters.  Although this feature is in Beta, it is a much better implementation of what I had in mind.

Thank you for your time, but I do not think this needs to stay open now.
